### ISSUE TITLE
Please use Vue for the front end side.

### ISSUE THREAD

**Issue by adityapurwa** [goFrendiAsgard/workshop-fullstack#1]
By using Vue, it would reduce the barrier of needing to setup React and its JSX compilation process.

With Vue, you can just `script src` it from a CDN and start hacking - remember that the workshop time is limited and the audience doesn't want to have their computer full of `node_modules`.

Remember, monoculture is dangerous and we need more job posting that list Vue as its requirements.

**Comment by goFrendiAsgard** [comment]
@fahmi-idris gimana Mi? Vue, react, atau dua-duanya?

**Comment by adityapurwa** [comment]
Please consider the risk of having webpack and babel installed on the lab's computer. Let alone setting up the webpack configuration and probably node version incompatibility

TL;DR - VUE FTW

**Comment by goFrendiAsgard** [comment]
Considering the time-frame, I don't think it is going to be that kind of workshop. But I'll make sure to at least mention `vue` as good alternative. And I guess if we still able to catch up with the deadline, either I or Fahmi, will try to implement the front-end with vue also.

Also, I'll be more than happy to accept your pull request or enlist you as contributor :)

**Comment by adityapurwa** [comment]
> Considering the time-frame, I don't think it is going to be that kind of workshop.

Exactly, it would be hard to setup React with webpack babel and stuff within the time-frame.

Vue can be set up in a minute.
